src/celestia/qt/qtappwin.cpp

```diff
@@ -401,12 +405,26 @@ void CelestiaAppWindow::init(const QString& qConfigFileName,
     connect(fullScreenAction, SIGNAL(triggered()), this, SLOT(slotToggleFullScreen()));
     viewMenu->addAction(fullScreenAction);
 
+    QMenu *fpsMenu = new QMenu(_("FPS rate"), this);
+    
+    std::vector<int> fps_array = { 0, 15, 30, 60, 120 };
```

Comment:
why vector instead of array? array is faster in this case as it's a tiny wrapper around c-style arrays.

Reply:
Perhaps due to my limited knowledge of C++11 standard. I never used std::array before and overlooked it in Your example. Thanks for review. Edited to use array.